### ISSUE TITLE
Support for other region endpoints

### ISSUE THREAD

**Issue by oflahero** [Microsoft/Cognitive-TextAnalytics-DotNet#5]
Hey there, nice job on the wrapper! Any chance of extra *Client constructors that take a URL endpoint parameter as well as the API key please? Currently we're just limited to West US. Thanks!

(I realise you can still just set the Url property of the *Client after construction, but a single constructor call would be cleaner/nice to have.)

**Comment by herecydev** [comment]
Yep totally agree here, it's completely unintuitive to apply some default (of which a minority of customers would fall under) and not indicate that you should be supply this alongside the key
